client/src/main/java/com/vesoft/nebula/client/graph/net/NebulaPool.java

```diff
@@ -32,19 +32,13 @@ public boolean init(List<HostAddress> addresses, NebulaPoolConfig config)
             throws UnknownHostException {
         List<HostAddress> newAddrs = hostToIp(addresses);
         this.loadBalancer = new RoundRobinLoadBalancer(newAddrs, config.getTimeout());
-        ConnObjectPool objectPool = new ConnObjectPool(this.loadBalancer, config);
-        this.objectPool = new GenericObjectPool<>(objectPool);
-        GenericObjectPoolConfig objConfig = new GenericObjectPoolConfig();
-        objConfig.setMinIdle(config.getMinConnSize());
+        GenericKeyedObjectPoolConfig objConfig = new GenericKeyedObjectPoolConfig();
         objConfig.setMaxTotal(config.getMaxConnSize());
-        objConfig.setMinEvictableIdleTimeMillis(
-                config.getIdleTime() <= 0 ? Long.MAX_VALUE : config.getIdleTime());
-        this.objectPool.setConfig(objConfig);
-
-        AbandonedConfig abandonedConfig = new AbandonedConfig();
-        abandonedConfig.setRemoveAbandonedOnBorrow(true);
-        this.objectPool.setAbandonedConfig(abandonedConfig);
-        return objectPool.init();
+        // objConfig.setTestOnBorrow(true);
```

Comment:
this is unnecessary ?

Reply:
yeah, it an option to test the object. For efficiency, this is not turned on by default